kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html

```diff
@@ -272,14 +272,20 @@
                 ng-click="metadataExpanded = !metadataExpanded">
             <span ng-hide="metadataExpanded">▸ Show</span>
             <span ng-show="metadataExpanded">▾ Hide</span>
-            full metadata
+            image metadata
```

Comment:
What's the motivation behind this change? Looking at the UI there is already metadata on display above this so show image metadata doesn't seem to make as much sense. I imagine that the objection of full is that there is more metadata not being displayed?

![image](https://user-images.githubusercontent.com/1236466/109641465-a4a46b80-7b49-11eb-9009-2221e577e01c.png)

Reply:
I agree with @sihil. If `full` is no good, maybe we can go with `more`?

Reply:
This change was inspired by a suggestion on the [proposal](https://github.com/guardian/grid/issues/3167) to change from "Show full metadata" to "Show image metadata".

Reply:
My suggestion makes no sense: `Hide more metadata` sounds idiotic. Akash, Alex and me think `additional` will work and is better than `full` (not really true).

Reply:
Changing the text to "Show additional metadata" and "Hide additional metadata". Thanks @paperboyo